robot_sdk/wrappers/ec_wrapper.py

```
@@ -67,43 +77,45 @@ def get_joint_angles(self) -> Optional[List[float]]:
             print(f"Error getting joint angles: {e}")
             return None
 
-    def set_hand_teaching_mode(self, enable: bool) -> Optional[bool]:
+    def set_hand_teach(self, enable: bool) -> Optional[bool]:
         '''
         Enable or disable hand teaching mode (hand dragging).
         '''
         try:
-            return self.client.set_hand_drag(switch=enable)
+            self.client.set_hand_drag(switch=enable)
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)